pkg/api/types.go

```diff
@@ -1913,3 +1913,11 @@ type SELinuxOptions struct {
 	// SELinux level label.
 	Level string `json:"level,omitempty" description:"the level label to apply to the container"`
 }
+
+type RangeAllocation struct {
+	TypeMeta   `json:",inline"`
+	ObjectMeta `json:"metadata,omitempty"`
+
+	Range string `json:"range"`
```

Comment:
For the sake of poor old Pete, please comment these?  What are they?  What are valid values?